crates/bevy_render/src/settings.rs

```diff
@@ -16,6 +16,8 @@ pub enum WgpuSettingsPriority {
 /// Provides configuration for renderer initialization. Use [`RenderDevice::features`](crate::renderer::RenderDevice::features),
 /// [`RenderDevice::limits`](crate::renderer::RenderDevice::limits), and the [`WgpuAdapterInfo`](crate::render_resource::WgpuAdapterInfo)
 /// resource to get runtime information about the actual adapter, backend, features, and limits.
+/// NOTE: [`Backends::GL`](Backends::GL) is disabled by default.
+/// You can use [`Backends::GL`](Backends::GL) on Windows only with the help of [`ANGLE`](https://github.com/gfx-rs/wgpu#angle).
```

Comment:
```suggestion
/// [`Backends::GL`](Backends::GL) only works on Windows via [`ANGLE`](https://github.com/gfx-rs/wgpu#angle).
```

Reply:
I committed this suggestion too, but now I think that it can be misunderstood as if all `WgpuSettings` require `ANGLE` on Windows 🤔

Reply:
Maybe it should be something like "GL is disabled by default and requires ANGL on Windows"?

Reply:
I think if we add a line break above the NOTE this will be clear by grouping.

Reply:
I didn't add a line break intentionally cause on docs.rs everything after first empty line is shown only on page of the item itself, not the module containing it, and docs.rs doesn't give any hints whether there are additional docs on an item, so I thought users can miss this rather important stuff. Dk if this is true and maybe I should add a line break anyway, your thoughts? 🤔 

Also from a fresh point of view I think that changing "this" to "it" solves ambiguity.

Reply:
I didn’t see this discussion and made my own suggestions. Let me know what you both think of them as to clarity / too much information.

Reply:
I'm happy with your suggestion :)